packages/client/src/__tests__/types/deprecation/test.ts

```diff
@@ -44,7 +44,7 @@ async function main() {
   const executeRaw1 = await prisma.$executeRaw`SELECT * FROM User WHERE id = 1`
   const executeRaw2 =
     await prisma.$executeRaw`SELECT * FROM User WHERE id = ${1}`
-  const executeRaw3 = await prisma.$executeRaw(
+  const executeRaw3 = await prisma.$executeRawUnsafe(
```

Comment:
Same here, I'm not sure of a reason to support more parameters since JS supports template strings.